nautobot/core/management/commands/populate_database.py

```diff
@@ -36,6 +43,20 @@ def handle(self, *args, **options):
             raise CommandError('Unable to load data factories. Is the "factory-boy" package installed?') from err
 
         if options["flush"]:
+            if options["interactive"]:
+                confirm = input(
+                    f"""\
+You have requested a flush of the database before generating new data.
+This will IRREVERSIBLY DESTROY all data in the "{connections[DEFAULT_DB_ALIAS].settings_dict['NAME']}" database,
+including all user accounts, and return each table to an empty state.
+Are you SURE you want to do this?
+
+Type 'yes' to continue, or 'no' to cancel:"""
```

Comment:
I know this is super pedantic but the UX on having the space after the prompt helps!

```suggestion
Type 'yes' to continue, or 'no' to cancel: """
```